abixen-platform-core/src/main/java/com/abixen/platform/core/application/service/impl/SecurityServiceImpl.java

```diff
@@ -83,7 +83,7 @@ public Boolean hasUserPermissionToObject(User user, PermissionName permissionNam
         }
 
         if (securableModel.getCreatedBy() != null && securableModel.getCreatedBy().getId().equals(user.getId())) {
-            List<AclEntry> ownerAclEntries = aclEntryRepository.findAll(permissionName, AclSidType.OWNER, 0L, AclClassName.getByName(securableModel.getClass().getCanonicalName()), securableModel.getId());
+            List<AclEntry> ownerAclEntries = aclEntryService.findAll(permissionName, AclSidType.OWNER, 0L, AclClassName.getByName(securableModel.getClass().getCanonicalName()), securableModel.getId());
             if (ownerAclEntries.size() > 0) {
                 if (log.isDebugEnabled()) {
                     log.debug("User " + user.getUsername() + " has permission " + permissionName + " to object " + securableModel.getClass().getCanonicalName() + "[id=" + securableModel.getId() + "] based on that he is the owner.");
```

Comment:
message from log.debug will be only displaing on console with debug level login. We need to invoke isDebugEnabled() ?